pages/news.py

```diff
@@ -16,6 +16,119 @@ def markdown_link(self):
+_("""
+Documents initially made public by the the Missouri police department
+describe their efforts on tracking Freenet usage.  Using a simple
+scheme, they claim achieving a near zero false positive rate for
+tracking the originator of a download.  While we applaud all public
+documentation on attacks, we have to point out that the claimed
+effectiveness of their attacks is based solely on flawed mathematics.
+In reality, the false positive rate of their method is at least 83%,
+and close to 100% in real world scenarios.
+
+The claimed effectiveness of their attack is based on a false
+assumption about the distribution of the HTL of incoming requests.  HTL (Hops
+to Live) is a number embedded in each request that is usually
+decremented when the request is forwarded, starting at a value of 18
+at the originator of the request, and serves to limit the number of
+hops a request can survive on the network.  As a security precaution,
+the HTL is decremented probabilistically on the first few hops.  In
+their tracking efforts, the Missouri police department assumes that
+when multiple requests for pieces of a file arrive from a single node,
+and all these requests have HTL 18 with no requests at HTL 17 or 16,
+this node must be the originator of the requests, with a false
+positive probability decreasing with the number of requests received
+on from the node.  This would be true if whether the HTL is
+decremented is decided per request, which has historically been the
+case in the early days of Freenet.  The vulnerability they claim to
+exploit has however been addressed in 2008 (last touched in [this
+commit][commithtl18]).  Whether the HTL is decremented is since
+decided per connection, so any probability they claim using the
+aforementioned method is false.
+
+For every connection there is a 50% chance that all the HTL18 requests
+(not only a single one, as would be the case with per-request
+probabilistic decrementing) did not originate from the node from which
+we received them but were forwarded one step. So for 10 connections
+(the lowest number of peers Freenet uses), there are on average 5
+other nodes whose HTL18 requests are forwarded with HTL18, so the
+probability that a given HTL18 request originated at the node from
+which we received it is only about 17% (1 in 6). If the node has more
+connections (normal are about 30), this probability is even lower. And
+this probability does not get higher when gathering more requests of
+chunks from a specific file or a specific kind of files, because they
+can reasonably all be forwarded from a different node — the one which
+really sent them.
+
+To get good statistics, an attacker would have to connect to the node
+under investigation over and over again at different times when the
+peers of the node changed.  This requires waiting at least 2 hours to
+change a significant number of peers — the only way to be sure would
+be to wait for the other node to go offline for more than 5 minutes
```

Comment:
I'd suggest shying from specific values like 5 minutes unless there's a specific reason to keep it.